js/client-teambuilder.js

```diff
@@ -1144,10 +1146,20 @@
 		},
 		validate: function () {
 			var format = this.curTeam.format;
+			var prerequisiteErrors = [];
 			if (!format) {
-				app.addPopupMessage('You need to pick a format to validate your team.');
+				prerequisiteErrors.push('- You need to pick a format to validate your team.');
+			}
```

Comment:
Wait, why is this necessary? If I remember correctly, the default validation is Anything Goes...